src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt

```diff
@@ -29,7 +29,7 @@ val RsFunction.isTest: Boolean
     get() {
         val stub = greenStub
         return stub?.isTest
-            ?: (queryAttributes.hasAtomAttribute("test") || queryAttributes.hasAtomAttribute("quickcheck"))
+            ?: (queryAttributes.hasMatchingAttribute(Regex("test")) || queryAttributes.hasAtomAttribute("quickcheck"))
```

Comment:
According to suggested changes in `attrsByRegex` and the fact that we don't want to accept any attribute with `test` substring in its name, let's change the pattern to `.*test`

Reply:
This change modifies property that used in stub index creation.
So we need to rebuild indices to take into account new implementation.
To do it, you need to increase `org.rust.lang.core.stubs.RsFileStub.Type#getStubVersion`.
Don't forget to rebase this branch before making the corresponding change to get the actual value of stub version